be/src/exec/exec_node.cpp

```diff
@@ -166,6 +166,9 @@ Status ExecNode::init_join_runtime_filters(const TPlanNode& tnode, RuntimeState*
     if (state != nullptr && state->query_options().__isset.runtime_filter_wait_timeout_ms) {
         _runtime_filter_collector.set_wait_timeout_ms(state->query_options().runtime_filter_wait_timeout_ms);
     }
+    if (state != nullptr && state->query_options().__isset.runtime_filter_scan_wait_time_ms) {
```

Comment:
Why scan is special, couldn't use `runtime_filter_wait_timeout_ms`?

Reply:
A GRF should try its best to take effects on ScanOperator, so ScanOperator should have a proper wait-time which different from other Operators. for now, runtime_filter_scan_wait_timeout is equivalent to runtime_filter_wait_timeout_ms, because a large value(60ms) have performance penalty on some queries. but in the future, ScanOperator's wait-time should be optimized and take the cost of ScanNode and the distance from HashJoinNode to ScanNode into consideration.